.github/workflows/ci.yaml

```diff
@@ -16,7 +16,26 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  unit_tests:
     uses: bazel-contrib/.github/.github/workflows/bazel.yaml@v5
     with:
       folders: '["."]'
+  bcr_test:
+    uses: bazel-contrib/.github/.github/workflows/bazel.yaml@v5
+    with:
+      folders: '["test/bcr_test"]'
+      exclude: |
+        [
+          {"bzlmodEnabled": false},
+          {"bazelversion": "5.4.0"}
+        ]
+  test:
+    runs-on: ubuntu-latest
+    needs:
+      - unit_tests
+      - bcr_test
+    if: ${{ always() }}
+    steps:
+      - uses: cgrindel/gha_join_jobs@v1
```

Comment:
Could you explain what this does?

Reply:
I presume that the rep is setup to ensure that  the `test` job succeeds. Instead of updating the GH repo settings to also check the new job,  I added a new job that ensures that all of the dependent jobs succeed. Now, when new jobs are added to the CI, we do not need to update the GH repo config.

Reply:
Luckily this repo is under bazel-contrib and it's not using Bazel CI, so I don't have any concerns.